influxql/token.go

```diff
@@ -54,8 +55,8 @@ const (
 	SEMICOLON // ;
 	DOT       // .
 
-	keyword_beg
-	// Keywords
+	keywordBeg
+	// ALL and other Keywords
```

Comment:
I don't think this comment adds anything useful. We can take out the comment line if it's complaining from golint.